test/tests/partitioners/single_rank_partitioner/single_rank_partitioner.i

```diff
@@ -0,0 +1,53 @@
+[Mesh]
```

Comment:
Why do we always do simple diffusion on this?

I say just do a boring test with
```
[Problem]
  solve = false
[]

[Executioner]
  type = Steady
[]